roles/kubernetes-apps/metrics_server/templates/metrics-server-deployment.yaml.j2

```diff
@@ -67,63 +67,19 @@ spec:
           failureThreshold: 3
           initialDelaySeconds: 40
         securityContext:
-          capabilities:
-            drop: ["all"]
-            add: ["NET_BIND_SERVICE"]
           readOnlyRootFilesystem: true
           runAsGroup: 10001
           runAsNonRoot: true
           runAsUser: 10001
+          allowPrivilegeEscalation: false
```

Comment:
All changes seems good for me according to https://github.com/kubernetes-sigs/metrics-server/blob/93b9ddc8cefa40d0fc7fc13ba741220a6a463bc0/manifests/base/deployment.yaml